pkg/cmd/codespace/jupyter.go

```diff
@@ -43,8 +44,14 @@ func (a *App) Jupyter(ctx context.Context, codespaceName string) (err error) {
 	}
 	defer safeClose(session, &err)
 
+	client, err := grpc.Connect(ctx, session, "")
+	if err != nil {
+		return fmt.Errorf("error connecting to internal server: %w", err)
+	}
+	defer safeClose(client, &err)
```

Comment:
@josebalius Just want to confirm that this is the correct way to call the `Close()` function

Reply:
Yep, perfect.